Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.py

```diff
@@ -0,0 +1,212 @@
+
+    return file_entry
+
+
+def get_indicators(client: Client, args: Dict[str, Any]) -> list:
+    serial = args.get('serial')
+    if serial:
+        api_query = "indicators?serial%5B%5D=" + serial
+    else:
+        max_time = datetime.now() - timedelta(hours=48)
+        api_query = "indicators?updated_after="
+        api_query = api_query + str(max_time)
+        api_query = api_query.replace(":", "%3A")
+
+    raw_response = client.api_request(api_query)
+    data = raw_response['indicators']
+    page_number = 2
+    full_response = raw_response
+
+    while raw_response['total_pages'] != raw_response['page']:
+        if serial:
+            api_query = "indicators?page=" + str(page_number) + "&serial%5B%5D=" + serial
+        else:
+            api_query = "indicators?page=" + str(page_number) + "&updated_after=" + str(max_time)
+            api_query = api_query.replace(":", "%3A")
+        page_number += 1
+        raw_response = client.api_request(api_query)
+        new_data = raw_response['indicators']
+
+        data.extend(new_data)
+        for item in new_data:
+            if item not in full_response['indicators']:
+                full_response['indicators'].append(item)
+
+    results = CommandResults(
+        outputs_prefix='Dragos.Indicators',
+        outputs_key_field='indicator_id',
+        outputs=data,
+        raw_response=full_response
+    )
+
+    return results
+
+
+def fetch_incidents(client: Client, last_run: dict, first_fetch: str) -> Tuple[list, dict]:
+    if last_run == {}:
+        last_fetch = None
+    else:
+        last_fetch = last_run.get('time')
+
+    if last_fetch is None:
+        last_fetch = dateparser.parse(first_fetch)
+    else:
+        last_fetch = dateparser.parse(last_fetch)
+
+    max_time = last_fetch
+    #current_time = datetime.strptime(time, '%Y-%m-%dT%H:%M:%S')
+    #triggered_time = '[{},)'.format(datetime.strftime(time, '%Y-%m-%d %H:%M:%S'))
+
+    api_query = "products?updated_after="
+    api_query = api_query + str(max_time)
+    api_query = api_query.replace(":", "%3A")
+    api_query = api_query.replace(" ", "%20")
+
+    incident_data = client.api_request(api_query)
+    incidents = []
+    items = incident_data['products']
+
+    for item in items:
+        item['updated_at'] = item['updated_at'][:-5]
+        incident_time = dateparser.parse(item['updated_at'])
+        incident = {
+            'name': item['title'],
+            'occurred': incident_time.strftime('%Y-%m-%dT%H:%M:%SZ'),
+            'rawJSON': json.dumps(item)
+        }
+
+        incidents.append(incident)
+
+        if incident_time > max_time:
+            max_time = incident_time
+
+    next_run = {'time': max_time.strftime('%Y-%m-%dT%H:%M:%S')}
+    incidents.reverse()
+
+    return incidents, next_run
+
+
+def main() -> None:
+    """Main method used to run actions."""
+    try:
+        demisto_params = demisto.params()
+        base_url = demisto_params.get("url", "").rstrip("/")
```

Comment:
please update here the usage of the `base url `according the comment in the`DragosWorldview.yml` file

Reply:
@crineacc 
after you removed the `api/v1` part from the base url 
you should append it **in the code** to the base_url
otherwise i think it will not work
does it work for you currently ?

Reply:
Yep you're correct I forgot to put that back into the base url so it is not currently working thanks for pointing that out.

Reply:
You're right, I forgot to make that corresponding change. It does not work at the moment